mkdocs.yml

```diff
@@ -6,139 +6,101 @@ theme:
-  - 'Configuration with OSG-Configure': 'other/configuration-with-osg-configure.md'
   - 'Registration': 'common/registration.md'
-- Other:
-  - 'Install GlideinWMS Frontend': 'other/install-gwms-frontend.md'
-  - 'Install a CVMFS Stratum 1': 'other/install-cvmfs-stratum1.md'
-  - 'Troubleshooting Gratia': 'other/troubleshooting-gratia.md'
-  - 'Install Transfer Log Filebeats': 'other/schedd-filebeats.md'
-- Release Information:
-  - 'Release Note Overview': 'release/notes.md'
-  - 3.5 Release Notes:
-    - 'OSG Release 3.5.30': 'release/3.5/release-3-5-30.md'
-    - 'OSG Release 3.5.29': 'release/3.5/release-3-5-29.md'
-    - 'OSG Release 3.5.28-2': 'release/3.5/release-3-5-28-2.md'
-    - 'OSG Release 3.5.28': 'release/3.5/release-3-5-28.md'
-    - 'OSG Release 3.5.27': 'release/3.5/release-3-5-27.md'
-    - 'OSG Release 3.5.26': 'release/3.5/release-3-5-26.md'
-    - 'OSG Release 3.5.25-2': 'release/3.5/release-3-5-25-2.md'
-    - 'OSG Release 3.5.25': 'release/3.5/release-3-5-25.md'
-    - 'OSG Release 3.5.24': 'release/3.5/release-3-5-24.md'
-    - 'OSG Release 3.5.23': 'release/3.5/release-3-5-23.md'
-    - 'OSG Release 3.5.22': 'release/3.5/release-3-5-22.md'
-    - 'OSG Release 3.5.21-2': 'release/3.5/release-3-5-21-2.md'
-    - 'OSG Release 3.5.21': 'release/3.5/release-3-5-21.md'
-    - 'OSG Release 3.5.20': 'release/3.5/release-3-5-20.md'
-    - 'OSG Release 3.5.19': 'release/3.5/release-3-5-19.md'
-    - 'OSG Release 3.5.18': 'release/3.5/release-3-5-18.md'
-    - 'OSG Release 3.5.17': 'release/3.5/release-3-5-17.md'
-    - 'OSG Release 3.5.16': 'release/3.5/release-3-5-16.md'
-    - 'OSG Release 3.5.15-3': 'release/3.5/release-3-5-15-3.md'
-    - 'OSG Release 3.5.15-2': 'release/3.5/release-3-5-15-2.md'
-    - 'OSG Release 3.5.15': 'release/3.5/release-3-5-15.md'
-    - 'OSG Release 3.5.14': 'release/3.5/release-3-5-14.md'
-    - 'OSG Release 3.5.13': 'release/3.5/release-3-5-13.md'
-    - 'OSG Release 3.5.12': 'release/3.5/release-3-5-12.md'
-    - 'OSG Release 3.5.11': 'release/3.5/release-3-5-11.md'
-    - 'OSG Release 3.5.10-2': 'release/3.5/release-3-5-10-2.md'
-    - 'OSG Release 3.5.10': 'release/3.5/release-3-5-10.md'
-    - 'OSG Release 3.5.9': 'release/3.5/release-3-5-9.md'
-    - 'OSG Release 3.5.8-4': 'release/3.5/release-3-5-8-4.md'
-    - 'OSG Release 3.5.8-3': 'release/3.5/release-3-5-8-3.md'
-    - 'OSG Release 3.5.8-2': 'release/3.5/release-3-5-8-2.md'
-    - 'OSG Release 3.5.8': 'release/3.5/release-3-5-8.md'
-    - 'OSG Release 3.5.7': 'release/3.5/release-3-5-7.md'
-    - 'OSG Release 3.5.6': 'release/3.5/release-3-5-6.md'
-    - 'OSG Release 3.5.5': 'release/3.5/release-3-5-5.md'
-    - 'OSG Release 3.5.4': 'release/3.5/release-3-5-4.md'
-    - 'OSG Release 3.5.3': 'release/3.5/release-3-5-3.md'
-    - 'OSG Release 3.5.2': 'release/3.5/release-3-5-2.md'
-    - 'OSG Release 3.5.1': 'release/3.5/release-3-5-1.md'
-    - 'OSG Release 3.5.0': 'release/3.5/release-3-5-0.md'
+  - Security:
```

Comment:
Throwing this out there: what if we get rid of the Security bullet and promote everything in it to Common? Security sounds like it's for making your site more secure if you care about that sort of thing, but installing CA certs and getting a host cert are pretty much mandatory -- it's about getting your site to work in the first place, not locking it down more.

Reply:
That's how I had it originally before tossing it back in on a whim so that's fine by me